index.js

```diff
@@ -66,6 +66,7 @@ function getBrowserRules() {
     [ 'yandexbrowser', /YaBrowser\/([0-9\._]+)/ ],
     [ 'vivaldi', /Vivaldi\/([0-9\.]+)/ ],
     [ 'kakaotalk', /KAKAOTALK\s([0-9\.]+)/ ],
+    [ 'samsung internet', /SamsungBrowser\/([0-9\.]+)/ ],
```

Comment:
What do you think about having this as just `samsung` instead of `samsung internet`?  Completely up to you though as I really don't have enough context around what Samsung are doing in the world of their browsers.

Reply:
@DamonOehlman On Samsung phones at least, I know that the default browser app is called Samsung Internet. And that's how Samsung's documentation refers to it. So I think it makes sense to leave it as `samsung internet`. Seems like that's what the browser is actually called.

Reply:
For the sake of standardization `samsung` sounds like a better fit. Observe that none of the browser ids have whitespaces in them.

Reply:
@5punk, @DamonOehlman I updated the code accordingly. Please let me know if there is anything else. Thanks!